impl/src/main/java/org/jboss/forge/furnace/container/guice/lifecycle/GuiceAddonLifecycleProvider.java

```diff
@@ -54,12 +57,14 @@ public void start(Addon addon) throws Exception
    {
       Set<Module> modules = new LinkedHashSet<>();
       modules.add(new GuiceContainerModule(furnace, addonRegistry, addon));
+      modules.add(binder -> binder.bindListener(Matchers.any(), new ServiceTypeListener(addonRegistry)));
```

Comment:
I hope it's fine to use inline modules for one-liners. Is it fine, according to the codestyle of JBoss?